modules/special-rules/pages/save.adoc

```diff
@@ -1,2 +1,14 @@
 = Save
 
+These units can beat the odds and escape destruction -- perhaps through exceptional armour, protective fields, or extreme skills and toughness.
+
+---
+
+Where your unit has the {save} ability:
+
+* Each time the unit suffers a hit, from any source, roll a die.
+On a roll of 4+, ignore the hit -- it causes no damage.
+
+NOTE: Where void shields and power fields are in effect, apply the {save} ability only to attacks that ignore these.
```

Comment:
This bit is now completely bass-ackwards. 🤣

There are no units with a Save and with void shields or power fields. This is a reminder that attacks that bypass void shields or power fields, for example Distortion Cannon or close combat hits, don't bypass Saves.

```suggestion
NOTE: Apply the {save} ability against all hits, even to attacks that ignore void shields or power fields.
```

Reply:
Okay, it's great when you catch this stuff as 'rules SME'. The original text seemed ambiguous/unclear to me, and it seems that while my new sentence may be clearer it is also wrong. :-D  This kind of thing will happen sometimes when I rework the text as 'technical writer', so catching these is an important part of your 'rules SME' role.

However, to be clear — does the Save ability apply to hits on void shields and power fields? I didn't check, but I rather thought that it does not? You suggestion implies that Save should apply to shields/fields also ("all hits").

Reply:
Revised version which should be less ambiguous:

```suggestion
NOTE: Attacks which ignore void shields or power fields do *not* ignore the {save} ability.
```